Menus/RadialMenu/Scripts/RadialMenuUI.cs

```diff
@@ -333,6 +338,15 @@ public void SelectionOccurred()
 		{
 			if (m_HighlightedButton != null)
 				m_HighlightedButton.button.onClick.Invoke();
+
+			if (buttonClicked != null)
+				buttonClicked();
+		}
+
+		void OnButtonHover()
```

Comment:
Is this set up in the inspector? If so, probably should be consistent with naming.